ads/contentad.js

```diff
@@ -35,9 +35,10 @@ export function contentad(global, data) {
   /* Pass Source URL */
   let sourceUrl = window.context.sourceUrl;
   if (data.url) {
-    const host = window.context.location.host;
+    const s = document.createElement('a');
```

Comment:
@jridgewell I think I got it. I updated the code to use parseUrl. Please let me know if anything further needs to be done to get this merge request approved.